Source/Urho3D/Graphics/DebugRenderer.cpp

```diff
@@ -505,7 +541,7 @@ void DebugRenderer::Render()
     graphics->SetBlendMode(lineAntiAlias_ ? BLEND_ALPHA : BLEND_REPLACE);
     graphics->SetColorWrite(true);
     graphics->SetCullMode(CULL_NONE);
-    graphics->SetDepthWrite(true);

```

Comment:
mb similar "fill" for navMesh ? lines on floor looks very simple and do not so solid?